task/sast-java-sec-check/0.1/sast-java-sec-check.yaml

```diff
@@ -9,28 +9,30 @@ metadata:
     tekton.dev/tags: "appstudio, hacbs, java"
   name: sast-java-sec-check
 spec:
+  description: >-
+    Static Java code security test.
   workspaces:
     - name: workspace
   results:
-    - description: Test output
+    - description: Tekton task test output.
       name: HACBS_TEST_OUTPUT
   params:
     - name: PATH_CONTEXT
-      description: The path to your source code
+      description: The path to your source code.
       type: string
       default: .
     - name: OUTPUT_FORMAT
-      description: the format of findsecbugs output
+      description: The format of findsecbugs output.
       type: string
       default: "sarif"
     - name: OUTPUT_ONLY_ANALYZE
       description: |
-        only analyze given classes and packages;
-        end with .* to indicate classes in a package, .- to indicate a package prefix
+        Only analyze given classes and packages;
```

Comment:
not sure what to do with these two lines (30-31) in terms of recommended changes. is this a task description too?

Reply:
what about: Analyze only given classes and packages.

?